### PR TITLE
fix: continue if font family doesn't match in `getFontDetails`

### PR DESCRIPTION
**Pull request by qwerzl**
Just a simple fix; this PR modifies `getFontDetails` so that it continues the loop instead of break if font family doesn't match because the input CSS might have font families other than the requested one. This fixes the "Could not produce font face declaration from adobe for font family Barlow Semi Condensed." error in the playground as I have multiple fonts in that kit.

### CODE REVIEW

File: src/css/parse.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const styleMap: Record<string, string> = {
 export function extractFontFaceData (css: string, family?: string): NormalizedFontFaceData[] {
   const fontFaces: NormalizedFontFaceData[] = []
 
-  for (const node of findAll(parse(css), node => node.type === 'Atrule' && node.name === 'font-face')) {
```
Comment (danielroe) [inline src/css/parse.ts:37]: I feel there must be another way to do this. I’ll take a look …

Comment (danielroe) [reply src/css/parse.ts:37]: okay, I think the problem here is that we mustn't assume that `font-family` comes first - we probably need somehow to validate the node before iterating over its children...

Comment (danielroe) [reply src/css/parse.ts:37]: pushed an implementation - what do you think?

Comment (qwerzl) [reply src/css/parse.ts:37]: @danielroe This is awesome. I think this is good to go.

**Review by danielroe — APPROVED** [review]

**Pull request merged**